processing/src/main/java/org/apache/carbondata/processing/newflow/constants/DataLoadProcessorConstants.java

```diff
@@ -33,4 +33,8 @@
   public static final String COMPLEX_DELIMITERS = "COMPLEX_DELIMITERS";
 
   public static final String DIMENSION_LENGTHS = "DIMENSION_LENGTHS";
+
```

Comment:
can you rename this class to a more meaningful name like `DataLoadOptions`. It will be exposed to user as data load options, right?